ImportExport/CadbaseLibrary/CdbsAuth.py

```diff
@@ -0,0 +1,45 @@
+''' This file contains one class for authorization on the CADBase platform '''
+
+import json
+from PySide import QtCore
+from PySide2 import QtNetwork
+import FreeCAD as app
+from CadbaseLibrary.CdbsEvn import g_param, g_api_login, g_content_type
+
+class CdbsAuth:
+    ''' class for getting a token to access the CADBase platform '''
+
+    def __init__(self, username, password):
+        app.Console.PrintMessage('Getting a new token, please wait.\n')
+
+        query = {'user': {'username': username, 'password': password}}
+
+        self.do_request(query)
+
+    def do_request(self, query):
+        req = QtNetwork.QNetworkRequest(QtCore.QUrl(g_api_login))
+
+        req.setRawHeader(b'Content-Type', g_content_type);
+
+        body = json.dumps(query).encode('utf-8')
+        self.nam = QtNetwork.QNetworkAccessManager()
+        reply = self.nam.post(req, body)
+        loop = QtCore.QEventLoop()
+        reply.finished.connect(loop.quit)
+        loop.exec_()
+        self.handle_response(reply)
+
+    def handle_response(self, reply):
+        global g_param
```

Comment:
This line is redundant since Python will look up for it recursively in the parent scopes. The line is needed only in the case that you do `g_param = something`.

Naming the variable with a name starting with `g_` should is explicit enough to tell that the variable is global.